invenio/b2share/modules/b2deposit/restful.py

```diff
@@ -216,13 +222,28 @@ def get(self, oauth, domain_name, **kwargs):
         record_list = []
         start = page_offset * page_size
         stop = start + page_size
+
         for record_id in record_ids[start : stop]:
             record_details = get_record_details(record_id, curr_user_email)
             record_list.append(record_details)
-        if stop < len(record_ids):
-            record_list.append('...') # continuation indicator
-        return record_list
 
+        if len(record_list) == 0:
```

Comment:
You are correct that these cases exist, but my code above has exactly the same effect and is much shorter. Less code is almost always better. I don't see the point of having 6 ifs when you can return the same result with just one.